apps/client/src/components/RegisterLookupSummary.tsx

```diff
@@ -14,23 +14,34 @@ import {
 import AddressLines from "./AddressLines";
 import ChangeAddressModal from "./Location/ChangeAddressModal";
 
-// @TODO: Convert this file to TS
-// @TODO replace unused props from RegisterLookupSummary in LocationResult
-const RegisterLookupSummary = ({
+// @TODO: can you make a props file for these kinds of props, @andre?
+type zoningPlanProps = {
+  name: string;
+};
+
+type RegisterLookupSummaryProps = {
+  addressFromLocation: any;
+  compact: boolean;
+  setActiveState: Function;
+  topic: any; // @TODO: replace with custom hooks
+};
+
+const RegisterLookupSummary: React.FC<RegisterLookupSummaryProps> = ({
   addressFromLocation,
   compact,
   setActiveState,
   topic: { slug, hasIMTR },
 }) => {
-  const sessionContext = useContext(SessionContext);
+  // @TODO: replace with custom topic hooks
+  const sessionContext = useContext<SessionDataType>(SessionContext);
   const address = addressFromLocation
     ? addressFromLocation
-    : sessionContext[slug].address;
+    : sessionContext[slug].address; // @TODO: replace with custom hooks
   const { restrictions, zoningPlans } = address;
```

Comment:
I believe if we type `Session` then address and zoningPlan's will all inherit their types.
Can you add a comment?
`// @TODO; type Session and/or Address and remove types of zoningPlan below`